src/helpers.test.js

```diff
@@ -312,6 +338,36 @@ describe('getPrettierOptions', () => {
   });
 });
 
+describe('getPrettierCliOptions', () => {
+  test('translates empty options', () => {
+    const expected = '';
+    const actual = getPrettierCliOptions({});
+
+    expect(actual).toEqual(expected);
+  });
```

Comment:
I don't think this case should ever occur, should it? We're always going to retrieve all options, right?